index.js

```diff
@@ -4,140 +4,219 @@ window.addEventListener("load", () => {
   const eqButton = document.querySelector(".eButton");
   const numDisplay = document.querySelector(".numDisplay");
   const opArray = ["add", "subtract", "multiply", "divide"];
-  let currentString = numDisplay.textContent;
+  let currentString = "";
+  //This should always have STRING data. numDisplay, despite the name, should also always have STRING data.
```

Comment:
You don't need to change this, but FYI code comments are usually left BEFORE the line they're talking about. I was slightly confused when I read these comments until I realized each one is talking about the line above.

Reply:
Updated. If that's the convention, I should probably start following it.